packages/package_info/example/test_driver/integration_test.dart

```diff
@@ -2,6 +2,8 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+// @dart = 2.9
```

Comment:
Which issue did you see that require opting out?

Reply:
There is an issue with flutter_driver being legacy: 

<img width="845" alt="Screen Shot 2021-01-07 at 4 42 03 PM" src="https://user-images.githubusercontent.com/10687576/103948141-4a69ca00-5107-11eb-8df3-41ea796ad66f.png">

I wasn't sure when we were planning on migrating test since `integration_test` isn't null safe yet either.

Reply:
ah - you right https://github.com/flutter/flutter/blob/master/packages/flutter_driver/pubspec.yaml#L13